### ISSUE TITLE
hanging, latest version.

### ISSUE THREAD

**Issue by elazzzzyy** [khast3x/h8mail#92]
### Checkbox

- [Y] I have thoroughly checked the answer is not in the [wiki](https://github.com/khast3x/h8mail/wiki).

### Env

* h8mail version: latest 2.5.3
* Python version: 3.8
* Operating System: parrotOS (not vm) & WSL ubuntu 20.4

### Description

Hanging on any search with no error

### What I Did

```
cd h8mail cd h8mail pip3 install -r requirments.txt cd .. 
h8mail -t example@live.com (tried gmail too)
vvvvvvvvvvvvv
removing dupes
targets:
example@live.com
^^^^^^^^^^
hangs with nothing after that
```

**Comment by ananthu10** [comment]
samr here

**Comment by khast3x** [comment]
Hello,

This is because the default API, **scylla.sh, is currently under heavy traffic**. h8mail has a built-in test to check if scylla is up. But in this case, scylla responds to the check but not the actual requests, making h8mail hang.  

#### Workaround
Until scylla is back, you can use `--skip` so h8mail can avoid using it, and [use an additional API](https://github.com/khast3x/h8mail/wiki/Using-APIs) instead, such as IntelX or emailrep for free, or the premium ones such as Snusbase or Leak-Lookup.   
[Here is a good video tutorial](https://www.youtube.com/watch?v=8GBfDXzrVSw) on that matter.

If you want to help scylla stay free and functional, consider [buying the creator a coffee](https://www.buymeacoffee.com/Eiw47ImnT) :coffee: .

Cheers

**Comment by khast3x** [comment]
It seems the built-in check does not timeout correctly on h8mail's end either, resulting in the hang.  

The timeout fix will be included in the next release.  

The timeout fix is already pushed on the next branch [here](https://github.com/khast3x/h8mail/tree/2.5.4) if you want to get in advance.  
You'll need to `git checkout` correctly.

The mentioned workaround above is still the only viable way to get results.

Cheers

**Comment by ananthu10** [comment]
thanks for info

**Comment by khast3x** [comment]
Hello,

Fixed with PR #97 .
Run `pip3 install --upgrade h8mail` to update h8mail to the latest version (2.5.4).

Cheers!
